src/pudl/output/sql/denorm_plants_utils_ferc1.sql

```diff
@@ -0,0 +1,5 @@
+-- Build a view of useful FERC Plant & Utility information.
+CREATE VIEW denorm_plants_utils_ferc1 AS
+SELECT *
+FROM plants_ferc1
+    INNER JOIN utilities_ferc1 USING(utility_id_ferc1);
```

Comment:
Why is it appropriate to use an `INNER JOIN` here?

Reply:
Because the original python function uses the default join operation for [`pd.merge()`](https://pandas.pydata.org/docs/reference/api/pandas.merge.html) which is "inner":

```python
pu_df = pd.merge(
        pd.read_sql("plants_ferc1", pudl_engine),
        pd.read_sql("utilities_ferc1", pudl_engine),
        on="utility_id_ferc1",
    )
```